src/vmod_accept.c

```diff
@@ -3,11 +3,19 @@
 #include <stdlib.h>
 #include <ctype.h>
 #include <errno.h>
+#include <string.h>
+
+#include <cache/cache.h>
+#include <vcl.h>
+
+#ifndef VRT_H_INCLUDED
```

Comment:
drop the #ifndefs

Reply:
Sure, I can do that, and I'll add a patch to this branch shortly.

I'd only included those at the suggestion of Dridi on another feature request I'd made in varnish-cache, and Dridi indicated that's how he ensured that the includes would work with both 5.x and 6.x.

Reply:
Ah, ifthat's for compatibility issue, I'll be happy to take them then